std/typecons.d

```diff
@@ -3011,11 +3011,6 @@ unittest
         inout(Object) foo() inout;
     }
     BlackHole!Foo o;
-
-    // Bugzilla 12464
```

Comment:
https://issues.dlang.org/show_bug.cgi?id=12464

We should probably add another regression test to replace this one.